src/py/flwr/client/message_handler/message_handler_test.py

```diff
@@ -142,7 +142,9 @@ def test_client_without_get_properties() -> None:
     actual_msg = handle_legacy_message_from_msgtype(
         client_fn=_get_client_fn(client),
         message=message,
-        context=Context(node_id=1123, node_config={}, state=RecordSet(), run_config={}),
+        context=Context(
+            run_id=1123, node_id=1123, node_config={}, state=RecordSet(), run_config={}
```

Comment:
nit: use a different number of `run_id` and `node_id`, otherwise tests won't fail if the two get mixed up